web/src/pages/index.tsx

```diff
@@ -24,7 +24,7 @@ function HomepageHeader() {
         <div className={styles.buttons}>
           <Link
             className="button button--secondary button--lg"
-            to="/docs/get-started">
+            to="/docs/about-vast">
             Get Started
```

Comment:
I find it a bit weird to have a docs page labelled "Get Started" if this button with the same name doesn't link to it. Can we rename the "Get Started" docs page?